sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -61,6 +78,17 @@ export interface AppConfigurationPutKeyValueOptionalParams extends coreHttp.Requ
     label?: string;
 }
 
+// @public
+export interface AppConfigurationPutLockOptionalParams extends coreHttp.RequestOptionsBase {
+    ifMatch?: string;
+    ifNoneMatch?: string;
+    label?: string;
+}
+
+// @public
+export interface ClearReadOnlyOptions extends Pick<AppConfigurationDeleteLockOptionalParams, Exclude<keyof AppConfigurationDeleteLockOptionalParams, 'label'>> {
+}
+
 // @public
 export interface ConfigurationSetting {
     // (undocumented)
```

Comment:
nit:  add doc comments for each property here

Reply:
I've got another issue opened for this (#5191). 

These are the auto-generated models so I'd like to figure out a non-annoying way of updating the comments w/o losing or repasting them each time.